pom.xml

```diff
@@ -55,7 +55,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <java.level>8</java.level>
         <groovy.version>2.4.7</groovy.version>
-        <jenkins.version>2.204.4</jenkins.version>
+        <jenkins.version>2.222.4</jenkins.version>
```

Comment:
this is to align with the BOM

Reply:
You'll also need to change the version we build/test against in the Jenkinsfile too.
The Jenkinsfile lists the versions (of Jenkins) we build and test against, and it's typically "null" (i.e. latest) plus the minimum (currently 2.204.4 but will need to change to 2.222.4).

Reply:
I've bumped jenkins.version to 2.303.3 in #897, including updating the Jenkinsfile and BOM.